internal/provider/resource_gitlab_project_badge_test.go

```diff
@@ -83,6 +85,7 @@ func testAccCheckGitlabProjectBadgeExists(n string, badge *gitlab.ProjectBadge)
 type testAccGitlabProjectBadgeExpectedAttributes struct {
 	LinkURL  string
 	ImageURL string
+	Name     string
 }
 
 func testAccCheckGitlabProjectBadgeAttributes(badge *gitlab.ProjectBadge, want *testAccGitlabProjectBadgeExpectedAttributes) resource.TestCheckFunc {
```

Comment:
You need to add an actual assertion here for the new `Name` attribute.

However, we've recently started to rethink this testing approach and rather concentrate on using `resource.TestCheck...()` functions in combination with import verification test steps.
See more details here: 

* https://github.com/gitlabhq/terraform-provider-gitlab/issues/1008

... which would get rid of these checking mechanisms. However, I  don't expect you to refactor - we can merge this nonetheless - just in case you want to take a stab at it :) Let me know! 🎉

Reply:
@timofurrer I dont understand if this need be change now

Reply:
Yes, this function needs an addition `if` statement much the like ones for `LinkURL` and `ImageURL` to verify if the `Name` actually matches.

On line 100 something along the lines of:

```go
if badge.Name != want.Name {
    return fmt.Errorf("got name %s; want %s", badge.Name, want.Name)
}
```

Reply:
Ok, fixed